test/functional/kernel/hyperplane/test-kernel-hyperplane-2D.cpp.in

```diff
@@ -173,6 +173,11 @@ using HipKernelHyperplaneExecPols =
 // no existing SYCL implementation of Hyperplane
 #endif  // RAJA_ENABLE_SYCL
 
+// Switch to control testing old vs new reducer interface
+using UseParamReduce = camp::list<std::true_type>;
```

Comment:
How about putting these in one of the more general test headers with such lists.